host-support/terminal-functions.sh

```diff
@@ -97,4 +97,37 @@ check_pidevice_generation() {
             exit 1
             ;;
     esac
+}
+
+read_config() {
+    if [ -f /etc/rpi-sb-provisioner/config ]; then
+        . /etc/rpi-sb-provisioner/config
```

Comment:
Might be worth considering using systemd's EnvironmentFile / PassEnvironment so we can head this error off before we even attempt to start the service.

If we don't want to use that mechanism then it might be worth using ConfigurationDirectory= and $CONFIGURATION_DIRECTORY here rather than hard-coding the /etc path

Reply:
Not in favour of that, as this path must also be used by non-systemd components (specifically, the configuration tool)